gubernator/github/classifier.py

```diff
@@ -253,15 +260,20 @@ def get_xrefs(comments, merged):
     return sorted(xrefs)
 
 
-def get_comments(events):
+def get_comments(events, comments=None):
     '''
     Pick comments and pull-request review comments out of a list of events.
     Args:
         events: a list of (event_type str, event_body dict, timestamp).
+        comments_prev: the previous output of this function.
     Returns:
         comments: a list of dict(author=..., comment=..., timestamp=...),
                   ordered with the earliest comment first.
     '''
+    if comments is None:
+        comments = {}
+    else:
+        comments = {c['id']: c for c in comments}
```

Comment:
Why do you throw away the rest?

Reply:
It doesn't. I'm converting it back from a list to a dictionary for the processing. I'll add a comment.